src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/utils/UserContext.kt

```diff
@@ -7,7 +7,7 @@ object UserContext {
   private val authToken_ = ThreadLocal<String>()
 
   @JvmStatic
-  var authToken: String
+  var authToken: String?
```

Comment:
this caused an exception when authToken was null (e.g. in health or swagger urls)